--- a/configure.ac	2017-12-14 19:34:48.000000000 +0100
+++ b/configure.ac	2017-12-11 15:43:40.000000000 +0100
@@ -84,6 +84,7 @@
 # Platform specific setup
 #
 #############################
+
 AC_CANONICAL_HOST
 case "${host_os}" in
     mingw*)
@@ -109,6 +110,9 @@
     powerpc-*-darwin*)
         OPENCL_LIBS=""
         ;;
+    *androideabi*)
+        AM_CONDITIONAL([ADD_RT], false)
+        ;;
     *)
         # default
         AM_CONDITIONAL([ADD_RT], true)
